src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -552,7 +552,7 @@ private static string GetNativeDllSubFolderName(out string ext)
             }
             else if (Platform.IsMacOS)
             {
-                folderName = "osx-x64";
+                folderName = "osx-" + processArch;
```

Comment:
Can you also update the comment on line 233 to include an example for osx-arm64?

Reply:
done